### PR TITLE
feat(Shoob): add presence

### PR DESCRIPTION
**Pull request by TranukaKami**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Review by theusaf — CHANGES_REQUESTED** [review]
* [x] Run `yarn format`
* [ ] Fix title to match [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)
* [x] Add proof of working (add a few screenshots of the presence status in Discord to your main comment)

File: websites/S/Shoob/metadata.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	],
+	"service": "Shoob",
+	"description": {
+		"en": "Waifu Bot?! Anime Card Game by Anime Soul Discord! Connect with all Anime communities today!",
+		"zh_CN": "外福机器人？ ！动漫灵魂不和谐动漫纸牌游戏！立即与所有动漫社区联系！",
+		"ga_IE": "Waifu Bot?! Cluiche Cárta Anime ag Anime Soul Discord! Déan teagmháil le gach pobal Anime inniu!",
+		"nl": "Waifu-bot?! Anime-kaartspel van Anime Soul Discord! Maak vandaag nog contact met alle Anime-gemeenschappen!",
+		"vi_VN": "Bot Waifu ?! Game bài Anime của Anime Soul Discord! Kết nối với tất cả cộng đồng Anime ngay hôm nay!"
+	},
+	"url": "shoob.gg",
```
Comment (DarkVillager) [inline websites/S/Shoob/metadata.json:29]: https://animesoul.com/ still exists?

**Review by EncryptedDev — COMMENTED** [review]
For static pages, use a Record instead

**Comment by EncryptedDev** [comment]
Resolve CI too

File: websites/S/Shoob/metadata.json (hunk 1 above)
Comment (TranukaKami) [reply websites/S/Shoob/metadata.json:29]: yea its not the same pages, it was shoob.gg first V.

Comment (DarkVillager) [reply websites/S/Shoob/metadata.json:29]: If the old one still works then don't delete the presence.

**Comment by EncryptedDev** [comment]
Split this PR for the consituent presences

File: websites/S/Shoob/metadata.json (hunk 1 above)
Comment (TranukaKami) [reply websites/S/Shoob/metadata.json:29]: the Animesoul site will undergo a full remake icon,address and mostly the Name of the Site , I think i had to delete the Presence before it becomes more confusing

**Pull request closed without merging**